### PR TITLE
Js tweening

### PR DESCRIPTION
**Pull request by eulertour**
## Motivation

Adds the ability to pass tween information to the WebGL frontend and implements it for a few animations. Also simplifies the code within `frame_server_impl.py` somewhat and changes js* and Js* names to webgl* and WebGL*

Depends on #955 

## Overview / Explanation for Changes

Passing tween information to the WebGL frontend allows for animating mobjects without making an RPC back to manim on each frame, which makes the animation much smoother.

## Oneline Summary of Changes

```
- Allow passing tween information to the WebGL frontend (:pr:`PR NUMBER HERE`)
```

## Testing Status

All existing tests pass.

## Acknowledgements
- [x] I have read the [Contributing Guidelines](https://docs.manim.community/en/latest/contributing.html)

## Reviewer Checklist
- [ ] Newly added functions/classes are either private or have a docstring
- [ ] Newly added functions/classes have [tests](https://github.com/ManimCommunity/manim/wiki/Testing) added and (optional) examples in the docs
- [ ] Newly added documentation builds, looks correctly formatted, and adds no additional build warnings
- [ ] The oneline summary has been included [in the wiki](https://github.com/ManimCommunity/manim/wiki/Changelog-for-next-release)

### CODE REVIEW

File: manim/__main__.py [hunk 1]
```diff
@@ -14,7 +14,8 @@
 
 try:
     from manim.grpc.impl import frame_server_impl
-except ImportError:
+except ImportError as e:
+    print(e)
```
Comment (naveen521kk) [inline manim/__main__.py:18]: don't print it.

**Review by naveen521kk — CHANGES_REQUESTED** [review]

File: manim/__main__.py (hunk 1 above)
Comment (naveen521kk) [reply manim/__main__.py:18]: use something like logging and set it to debug mode or just use `pass` here.

Comment (eulertour) [reply manim/__main__.py]: I changed it to info. People aren't likely to try printing in debug mode if manim doesn't start.

Comment (naveen521kk) [reply manim/__main__.py]: That's optional dependency right? If yes, then it should be debug, if no info is ok.

Comment (eulertour) [reply manim/__main__.py]: Really it just doesn't make much sense to have this here. I moved it to the place before it's used so it'll only be printed if the flag is passed.

File: manim/__main__.py [hunk 2]
```diff
@@ -77,10 +72,10 @@ def main():
     else:
         config.digest_args(args)
         input_file = config.get_dir("input_file")
-        if config["use_js_renderer"]:
+        if config["use_webgl_renderer"]:
             try:
-                if frame_server_impl is None:
-                    raise ImportError("Dependencies for JS renderer is not installed.")
+                from manim.grpc.impl import frame_server_impl
+
```
Comment (naveen521kk) [inline manim/__main__.py:78]: I would like to have this error caught and the user informed that there are missing dependency and also a way to install then.

File: manim/_config/utils.py [hunk 3]
```diff
@@ -1027,22 +1030,28 @@ def dry_run(self, val: bool) -> None:
...
             self["disable_caching"] = True
 
-    js_renderer_path = property(
-        lambda self: self._d["js_renderer_path"],
-        lambda self, val: self._d.__setitem__("js_renderer_path", val),
+    webgl_renderer_path = property(
+        lambda self: self._d["webgl_renderer_path"],
+        lambda self, val: self._d.__setitem__("webgl_renderer_path", val),
         doc="Path to JS renderer.",
     )
```
Comment (naveen521kk) [inline manim/_config/utils.py:1047]: using properties here will look better cc @friedkeenan

**Review by naveen521kk — CHANGES_REQUESTED** [review]

File: manim/__main__.py (hunk 2 above)
Comment (eulertour) [reply manim/__main__.py:78]: The ModuleNotFoundError is already printed; there's no reason to rephrase what it's already saying.

File: manim/_config/utils.py (hunk 3 above)
Comment (eulertour) [reply manim/_config/utils.py:1047]: If we do decide that's better it should wait for a later PR; the other changes here are much more important and this is already done with the other attributes.

File: manim/__main__.py (hunk 2 above)
Comment (naveen521kk) [reply manim/__main__.py:78]: But it doesn't say how to install it though.  I think this should be done if we are doing something as optional. We should inform users on Import error that run
```sh
pip install manim[webgl_renderer]
```
to install the dependencies needed to run webgl renderer.

**Comment by eulertour** [comment]
I'll probably merge this tomorrow or the day after.

**Review by naveen521kk — APPROVED** [review]

**Pull request merged**